docs/pages/router/reference/middleware.mdx

```diff
@@ -189,6 +317,7 @@ export default middleware;
 - Middleware runs exclusively on the server and only for HTTP requests. It does not execute during client-side navigation, for example, with [`<Link />`](/versions/latest/sdk/router/#link) or native app screen transitions.
 - The request object passed to middleware is [immutable](#request-immutability) to prevent side effects. You cannot modify headers or consume the request body, ensuring it remains available for route handlers.
 - You can only have one root-level **+middleware.ts** in your app.
+- Matcher patterns must always begin with `/`.
```

Comment:
Is this a limitation or intended behaviour? if it's the former, wouldn't we normalise and mention that all matchers are relative to the root (`/`), or is this a requirement?